test/libsolidity/syntaxTests/errors/panic_reserved_name.sol

```diff
@@ -0,0 +1,5 @@
+// TODO: What if an error is imported and alias as Panic?
+// TODO I Think the best way would be to have Error in the global scope.
+error Panic(bytes2);
+// ----
+// SyntaxError 1855: (131-151): The built-in errors "Error" and "Panic" cannot be re-defined.
```

Comment:
I saw that you removed these TODOs from #10864 so I thought you made a decision here already.

If this still needs discussion, maybe it would be a good topic for the call tomorrow?

Reply:
I actually don't see a real alternative to what we have here.